hibernate-core/src/main/java/org/hibernate/engine/spi/ActionQueue.java

```diff
@@ -1129,9 +1127,6 @@ private boolean hasParent(BatchIdentifier batchIdentifier, List<BatchIdentifier>
 			}
 		}
 
-		// the mapping of entity names to their latest batch numbers.
-		private List<BatchIdentifier> latestBatches;
-
```

Comment:
The above field should be put into method as temporary variable rather than a field

Reply:
I totally agree with this one, could you extract it into a separate patch?

Reply:
What did you mean? Creating a separate git commit in this PR?

Reply:
yes, either a separate git commit or a separate PR, so to be able to backport this fix only (and only this), in case of need.

No need to have a separate PR if most other changes are undone.

Reply:
Now after many reverting, seems not much code change is left. Maybe we can keep it as it is now?

Reply:
done. Now a separate git commit is created for this change exlusively.